codehawk.properties

```diff
@@ -1,20 +1,20 @@
 category=External Analysers
 description=Analyze Java code smell
 homepageUrl=https://github.com/SDPMLab/CodeHawk
-archivedVersions=1.3
-publicVersions=1.4
+archivedVersions=1.4
+publicVersions=1.5
 
 defaults.mavenGroupId= org.codehawk.plugin
 defaults.mavenArtifactId= codehawk
 
+1.5.description= Added new rules
+1.5.sqVersions=[7.9,8.4.*]
```

Comment:
Do you really mean to keep users of more current versions from using your plugin? The current version is 8.6 (with 8.6.1 on the way). Best practice would end this range with `LATEST` to stay compatible with newest releases.